tests/integration/tables/file.cpp

```diff
@@ -208,17 +210,18 @@ TEST_F(FileTests, test_sanity) {
     ASSERT_EQ(row.at("filename"), test_file_name);
 
     if (isPlatform(PlatformType::TYPE_WINDOWS)) {
-      auto link_path = boost::filesystem::path(expected_path);
-
-      if (row.at("path").rfind(".lnk") != std::string::npos) {
```

Comment:
This was a bug in the test that would always fail so the shortcut functionality was never tested (because the loop iterates only through the `kFilenameList` and not the shortcut files.